arm-datalake-analytics/job/2015-11-01-preview/swagger/job.json

```diff
@@ -585,21 +586,25 @@
         },
         "totalCompilationTime": {
           "type": "string",
+          "format": "non-iso-duration",
```

Comment:
No it doesn't but I wanted to keep a format line in here for myself so that I know that these fields are not yet returning a supported ISO format on the server. This results in the generators treating these as strings, which is the current behavior.